packages/clerk-js/src/ui/elements/PhoneInput/PhoneInput.tsx

```diff
@@ -180,14 +179,22 @@ const PhoneInputBase = forwardRef<HTMLInputElement, PhoneInputProps & { feedback
           sx={[
             t => ({
               boxShadow: 'none',
-              borderRadius: t.radii.$md,
               height: '100%',
-              borderTopLeftRadius: '0',
-              borderBottomLeftRadius: '0',
-              paddingLeft: `${`+${selectedCountryOption.country.code}`.length + 1.5}ch`,
               transitionProperty: t.transitionProperty.$common,
               transitionTimingFunction: t.transitionTiming.$common,
               transitionDuration: t.transitionDuration.$focusRing,
+              // This ensures that this input will never have any border or boxShadow styles.
+              '&[type=tel]': {
+                borderRadius: t.radii.$md,
+                borderWidth: 0,
+                borderTopLeftRadius: 0,
+                borderBottomLeftRadius: 0,
+                paddingLeft: t.space.$1,
+                '&:focus': {
```

Comment:
What about hover and active states?

Reply:
Hover and active states are not changed by default but we can include them as well